### DISCUSSION TITLE
增加光影和材质包的资源

### DISCUSSION THREAD

**Discussion by 1085zgr** [Meloong-Git/PCL#809] [提问] [answered]
### 光影和材质包又十分难找，不如加入到下载页面里，简单便捷****

**Comment by WForst-Breeze** [comment, marked as answer]
光影： #396 由于下载源问题，不会制作
材质包、地图： #44 🧐

**Reply by WForst-Breeze** [reply]
这些资源几乎都可以在 Curseforge 上找到的（部分光影可能在这里找不到），就是访问速度可能稍微慢一点，可以看看我在[ 这里 ](https://github.com/Hex-Dragon/PCL2/issues/776#issuecomment-1368895965)给出的解决方案
